rust/feature-flags/src/router.rs

```diff
@@ -26,9 +39,30 @@ where
         geoip,
     };
 
-    Router::new()
+    let status_router = Router::new()
+        .route("/", get(index))
+        .route("/_readiness", get(index))
+        .route("/_liveness", get(move || ready(liveness.get_status())));
+
+    let flags_router = Router::new()
         .route("/flags", post(v0_endpoint::flags).get(v0_endpoint::flags))
-        .with_state(state)
+        .layer(ConcurrencyLimitLayer::new(concurrency))
```

Comment:
Why add this?

Reply:
The config had a "concurrency" value, I figured the idea was to limit how many requests the pod is concurrently processing, and this is how you do that. Generally if requests _can_ result in significant memory usage, limiting how many you're processing at once is a good move.